pages/k8s/upgrading.md

```diff
@@ -0,0 +1,319 @@
+---
+title: "Upgrading | Canonical Distribution of Kubernetes&reg;"
+keywords: juju, upgrading, track, version
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: upgrading.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+
+# Upgrading
+
+It is recommended that you keep your **Kubernetes** deployment updated to the
+latest available stable version. You should also update the other applications
+which make up the **Canonical Distribution of Kubernetes<sup>&reg;</sup>.**
+Keeping up to date makes sure you have the latest bug-fixes and security
+patches for smooth operation of your cluster.
+
+There is no set cadence to the releases of **Kubernetes**,  but you can check
+the latest release version on the  
+[Kubernetes release page on GitHub][k8s-release].  The **CDK** is kept in close sync
+with upstream Kubernetes.
+
+<div class="p-notification--information">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+<strong>Kubernetes</strong> will automatically handle patch releases. This means that
+the cluster will perform an unattended automatic upgrade between patch versions, e.g.
+1.10.7 to 1.10.8. Attended upgrades are only required when you wish to upgrade a minor
+version, e.g. 1.9.x to 1.10.x.
+  </p>
+</div>
+
+You can see which version of each application is currently deployed by running
+
+```bash
+juju status
+```
+
+The 'App' section of the output lists each application and its version number.
```

Comment:
maybe could use a screenshot to show the parts of the output mentioned below?

Reply:
Yeah that would be good. Do you need one provided?

Reply:
No, this was more a note for myself, I can take a screenshot ;)